mlrun/model_monitoring/db/tsdb/tdengine/tdengine_connector.py

```diff
@@ -260,13 +326,65 @@ def read_predictions(
         start: datetime,
         end: datetime,
         aggregation_window: typing.Optional[str] = None,
+        limit: typing.Optional[int] = None,
     ) -> typing.Union[
         mm_schemas.ModelEndpointMonitoringMetricValues,
         mm_schemas.ModelEndpointMonitoringMetricNoData,
     ]:
-        raise NotImplementedError
+        if not aggregation_window:
+            logger.warning(
+                "Aggregation window is not provided, defaulting to 10 minute."
```

Comment:
```suggestion
                "Aggregation window is not provided, defaulting to 10 minutes."
```

Reply:
Also, that's inconsistent with the behvaiour of the TSDB implementation, isn't it?

Reply:
I'll remove it